src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -295,3 +295,6 @@ export const getHomelessOrAtRisk = formData => {
     homelessStatus === HOMELESSNESS_TYPES.atRisk
   );
 };
+
+export const isAnsweringPtsdForm = formData =>
+  _.get('view:uploadPtsdChoice', formData, '') === 'answerQuestions';
```

Comment:
This prompted me to look more closely at how we're distinguishing between the 781 and 781a flow.

Turns out, we need to change the names so they're different between the two. :grimacing: I filed a bug report for it [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15194).

I'm not sure whether this should be changed now or in the bug fix PR, but it will need to get changed up to reflect whether the veteran selected to answer the 781 or 781a.

Reply:
Alternatively, you could fix the bug in this PR if that'll save some headache. It's probably only another handful of LoCs. 

Don't make no nevermind to me either way, 's long as it gets fixed! :smile:

Reply:
I nested under 'other.' Thank you, @cvalarida ! It works great!